ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -329,14 +384,15 @@ object BigDiffy extends Command {
       s"""BigDiffy - pair-wise field-level statistical diff
         |Usage: ratatool $command [dataflow_options] [options]
         |
-        |  --mode=[avro|bigquery]
-        |  --key=<key>            '.' separated key field
-        |  --lhs=<path>           LHS File path or BigQuery table
-        |  --rhs=<path>           RHS File path or BigQuery table
-        |  --output=<output>      File path prefix for output
-        |  --ignore=<keys>        ',' separated field list to ignore
-        |  --unordered=<keys>     ',' separated field list to treat as unordered
-        |  [--with-header]        Output all TSVs with header rows
+        |  --input-mode=(avro|bigquery)     Diff-ing Avro or BQ records
+        |  [--output-mode=(gcs|bigquery)]   Saves to a text file in GCS or a BigQuery dataset
```

Comment:
what do the braces connote - optional args?

Reply:
Yea, common convention in bash help/usage guide